src/Utilities/TagsModal.js

```diff
@@ -8,13 +8,11 @@ import debounce from 'lodash/debounce';
 import flatten from 'lodash/flatten';
 
 const TagsModal = ({ filterTagsBy, onToggleModal, onApply, getTags }) => {
+  const PAGINATION_DEFAULT = { perPage: 10, page: 1 };
```

Comment:
Nice! We can even move this out of react component context so that is not reinitialised on every rerender.